webpack/JobInvocationDetail/JobInvocationConstants.js

```diff
@@ -29,3 +34,63 @@ export const DATE_OPTIONS = {
   hour12: false,
   timeZoneName: 'short',
 };
+
+const getColumnsStatus = ({ hostJobStatus, jobStatus }) => {
+  const defaultTitle = jobStatus === 'queued' ? __('Scheduled') : __('Unknown');
+
+  switch (hostJobStatus) {
+    case 'success':
+      return { title: __('Succeeded'), status: 0 };
+    case 'error':
+      return { title: __('Failed'), status: 1 };
+    case 'planned':
+      return { title: __('Scheduled'), status: 2 };
+    case 'running':
+      return { title: __('Pending'), status: 3 };
+    case 'cancelled':
+      return { title: __('Cancelled'), status: 4 };
+    default:
+      return { title: defaultTitle, status: 5 };
+  }
+};
+
+export const columns = {
+  name: {
+    title: __('Name'),
+    wrapper: ({ name }) => <a href={`/new/hosts/${name}`}>{name}</a>,
```

Comment:
Can the link to host be setting aware, example: https://github.com/theforeman/foreman_openscap/pull/571